internal/xds/bootstrap/bootstrap.go

```diff
@@ -154,15 +155,23 @@ func GetRenderedBootstrapConfig(opts *RenderBootsrapConfigOptions) (string, erro
 			}
 
 			// skip duplicate sinks
-			addr := fmt.Sprintf("%s:%d", sink.OpenTelemetry.Host, sink.OpenTelemetry.Port)
+			var host string
+			var port uint32
+			if sink.OpenTelemetry.Host != nil {
+				host, port = *sink.OpenTelemetry.Host, uint32(sink.OpenTelemetry.Port)
+			}
+			if len(sink.OpenTelemetry.BackendRefs) > 0 {
```

Comment:
maybe a TODO here to support multiple backendRefs in the future

Reply:
EG support multi sinks, I don't think that future exists.